src/OSPSuite.Core/Domain/Services/QuantityValuesUpdater.cs

```diff
@@ -89,11 +90,20 @@ private IParameter getOrAddModelParameter(ModelConfiguration modelConfiguration,
          if (parentContainer == null)
             return null;
 
-         parameter = _parameterFactory.CreateParameter(pathAndValueEntity.Name, dimension: pathAndValueEntity.Dimension, displayUnit: pathAndValueEntity.DisplayUnit)
-            .WithParentContainer(parentContainer);
+         var dimension = pathAndValueEntity.Dimension;
+         var name = pathAndValueEntity.Name;
+         var displayUnit = pathAndValueEntity.DisplayUnit;
+         var distributionType = pathAndValueEntity.DistributionType;
+         var value = pathAndValueEntity.Value;
 
-         simulationConfiguration.AddBuilderReference(parameter, pathAndValueEntity);
+         //if the distribution is undefined or the value is set, we create a default parameter to ensure that the value will take precedence.
+         //Otherwise, we create a distributed parameter and assume that required sub-parameters will be created as well
+         parameter = distributionType == null || value!=null ? 
+            _parameterFactory.CreateParameter(name, dimension: dimension, displayUnit: displayUnit) : 
+            _parameterFactory.CreateDistributedParameter(name, distributionType.Value, dimension: dimension, displayUnit: displayUnit);
```

Comment:
I wouldn't use named arguments if the argumet name is the same as the parameter name (`dimension: dimension` etc.)